codes/classical/bits/single_parity_check.yml

```diff
@@ -0,0 +1,55 @@
+#######################################################
```

Comment:
Please integrate into existing entry in next Github pull request. Somehow GIthub did not pick up this file as an edited version of the existing file, and wants me to manually make changes before merging. Maybe a fresh pull will fix this.